lib/msgpax.ex

```diff
@@ -33,7 +33,7 @@ defmodule Msgpax do
 
   <sup>2</sup>NaN and ±infinity are not enabled by default. See `unpack/2` for for more information.
 
-  <sup>3</sup>To deserialize back to `Msgpax.Bin` structs see the `unpack/2` options.
+  <sup>3</sup>To de-serialize back to `Msgpax.Bin` structs see the `unpack/2` options.
```

Comment:
```suggestion
  <sup>3</sup>To deserialize back to `Msgpax.Bin` structs see the `unpack/2` options.
```
Thanks for noticing that. 👍
Though I've decided to standardize on "deserialize".

Reply:
Done.